lib/routes/account.js

```diff
@@ -1027,10 +1027,20 @@ module.exports = function (
             reply(deviceArray.map(function (device) {
               if (! device.name) {
                 device.name = devices.synthesizeName(device)
+                log.event('deviceName:synthesize', request, {
+                  uid: uid,
+                  id: device.id,
+                  name: device.name
+                })
               }
 
               if (! device.type) {
                 device.type = device.uaDeviceType || 'desktop'
+                log.event('deviceType:synthesize', request, {
```

Comment:
This strikes me as the wrong sort of event - IIUC we only want to log these for our own metrics purposes, but `log.event` is specifically for events that go out via SNS/SQS for consumption by other serivces like sync or basket.  A `log.info` may be more appropriate here.